packages/tests-unit/tests/adapters/cache.test.ts

```diff
@@ -1,4 +1,6 @@
-import { hasCacheExtension } from "@opennextjs/aws/adapters/cache.js";
+/* eslint-disable sonarjs/no-duplicate-string */
+import S3Cache, { hasCacheExtension } from "@opennextjs/aws/adapters/cache.js";
```

Comment:
The import is not correct anymore

Reply:
In what sense? I couldn't see any meaningful differences that would make the import incorrect: https://github.com/opennextjs/opennextjs-aws/compare/178ab2b1c95701a5f20aec107acf1fe1c6e3d9be...main#diff-c8326fdbbc9eb7b098d401bf8245a207a236441bae48edf1597f313afd7fa1c6

Reply:
My bad this is correct